### PR TITLE
[WIP] Subtask/as 79 extra conditions tests b

### PR DESCRIPTION
**Pull request by swdpcomputing**
## Ticket Number

AS-79

## Description of change

Completed e2e tests for AS-79
## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by karunak21 — COMMENTED** [review]
**Line 6 -** 'The', should be in lower case 'the' to be consistent with the next lines
'page' is presented -> need not specify 'page'
Given the Householder planning appeal questionnaire tasks are presented
When user selects to provide any extra conditions
Then user should be presented with the opportunity to provide any extra conditions

'And'-> this should be verified in the above Then step in the glue code

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature [hunk 1]
```diff
@@ -0,0 +1,64 @@
+Feature: Appeals Extra Conditions
+  As a beta LPA I want to inform the Planning Inspectorate of any extra conditions I wish to be attached to the appeal decision (over and above the standard ones)
```
Comment (karunak21) [inline lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:2]: Have you verified the Standard conditions text

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  And the user remains on extra conditions page
+
+Scenario: AC06 - user can see the Appeal details side panel
+  Given user is in the extra conditions page
+  Then the appeal details panel is displayed on the right hand side of the page
+  And the user remains on extra conditions page
+
+Scenario: AC07 - Back link
+  Given user is in the extra conditions page
+  When user selects the back link  Then user navigates to the Task List
```
Comment (karunak21) [inline lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:58]: Then should be in the next line

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature [hunk 3]
```diff
@@ -0,0 +1,64 @@
+Feature: Appeals Extra Conditions
+  As a beta LPA I want to inform the Planning Inspectorate of any extra conditions I wish to be attached to the appeal decision (over and above the standard ones)
+So that the Inspector can decide whether or not to attach them to their decision.
+
+Scenario: AC01 - User navigates to 'Do you have extra conditions' page
+  Given The Householder planning appeal questionnaire page is presented
+  When the user selects the link 'Do you have any extra conditions?'
+  Then the user is presented with the 'Do you have any extra conditions?' page
+  And the Page title is 'Do you have any extra conditions? - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK'
```
Comment (karunak21) [inline lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:9]: Have you verified the Standard conditions text

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature [hunk 4]
```diff
@@ -0,0 +1,64 @@
+Feature: Appeals Extra Conditions
+  As a beta LPA I want to inform the Planning Inspectorate of any extra conditions I wish to be attached to the appeal decision (over and above the standard ones)
+So that the Inspector can decide whether or not to attach them to their decision.
+
+Scenario: AC01 - User navigates to 'Do you have extra conditions' page
+  Given The Householder planning appeal questionnaire page is presented
```
Comment (karunak21) [inline lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:6]: Given the householder planning appeal questionnaire task list is presented

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature (hunk 2 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:58]: Done.

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature (hunk 4 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:6]: Edited and matching step function edited.

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature (hunk 3 above)
Comment (swdpcomputing) [reply lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:9]: Yes

File: lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  Then the appeal details panel is displayed on the right hand side of the page
+  And the user remains on extra conditions page
+
+Scenario: AC07 - Back link
+  Given user is in the extra conditions page
+  When user selects the back link
+  Then user navigates to the Task List
+  And any information they have entered will not be saved
+
+Scenario: AC08 - User has completed the extra conditions page and returns to that page from the Task List
```
Comment (karunak21) [inline lpa-submissions-e2e-tests/cypress/integration/appealsExtraConditions.feature:62]: AC08 is not mentioned in Story descriptio

**Review by karunak21 — APPROVED** [review]

**Pull request merged**